addon/services/ajax.js

```diff
@@ -225,6 +227,7 @@ export default Ember.Service.extend({
    @return {Object | DS.AdapterError} response
  */
  handleResponse(status, headers, payload) {
+   payload = payload || {};
```

Comment:
I'm not 100% sure why, but when I was testing the `5XX` errors against a few different values, only the `510